src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -1051,16 +1062,35 @@ public FormValidation doCheckServerUrl(
         @RequirePOST
         @SuppressWarnings("unused") // used by jelly
         public ListBoxModel doFillCredentialsIdItems(
-                @AncestorInPath ItemGroup owner, @QueryParameter String serverUrl) {
-            checkPermission((owner instanceof AccessControlled ? (AccessControlled) owner : Jenkins.get()));
+                @AncestorInPath AccessControlled owner, @QueryParameter String serverUrl) {
```

Comment:
Not sure why you are changing this method. Did you intend to replace `checkPermission` usages with the new `checkManagePermission` method? And if so, there are a few other validation methods to update.